app/views/macros/search-results.njk

```diff
@@ -11,7 +11,7 @@
         <div class="govuk-grid-column-one-half">
           <p></P>
         </div>
-        <div class="govuk-grid-column-one-half">
+        <div class="govuk-grid-column-one-half" style="display: flex; justify-content: flex-end">
```

Comment:
Please add the style as css class within applications.scss file and reference the class here.

Reply:
agreed , added to applications.scss file now.